.github/workflows/sdk-regression.yml

```diff
@@ -34,6 +34,15 @@ jobs:
       - uses: xt0rted/pull-request-comment-branch@v2
         if: ${{ github.event.issue.pull_request }}
         id: comment-branch
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match
+        with:
+          text: ${{ steps.comment-branch.outputs.head_ref }}
+          regex: '^[a-zA-Z0-9_/\-]+$'
```

Comment:
is regex tested?
![Uploading Screenshot 1945-02-06 at 12.51.09 PM.png…]()

Reply:
<img width="1173" alt="Screenshot 1945-02-06 at 12 51 09 PM" src="https://user-images.githubusercontent.com/51262163/234501486-66b25be4-1502-4ea0-aeb8-a8b26486f591.png">

Reply:
We don't need to escape / for JS regex. On regexr:
<img width="328" alt="Screenshot 2023-04-26 at 5 27 31 PM" src="https://user-images.githubusercontent.com/2987087/234568081-d251834b-50b5-4df6-8966-6519e815ee80.png">